app/assets/javascripts/leaflet.map.js

```diff
@@ -55,6 +55,13 @@ L.OSM.Map = L.Map.extend({
 
     this.dataLayer = new L.OSM.DataLayer(null);
     this.dataLayer.options.code = 'D';
+    
+    this.gpsLayer = new L.OSM.GPS({
+      attribution: copyright,
+      code: "G",
+      keyid: "gps",
```

Comment:
Good question and I believe the answer is no, because attribution is only shown for base layers.